src/UI/Dependency/EnvironmentModifier.test.tsx

```diff
@@ -17,28 +19,32 @@ const DummyComponent: React.FC<{
   );
 };
 
-function setup() {
+function setup(definition: DefinitionMap) {
   const environmentId = "env";
   const store = getStoreInstance();
   store.dispatch.environment.setSettingsData({
     environment: environmentId,
     value: RemoteData.success({
       settings: {},
-      definition: EnvironmentSettings.definition,
+      definition,
     }),
   });
   const environmentModifier = EnvironmentModifierImpl();
   environmentModifier.setEnvironment(environmentId);
   const component = (
-    <StoreProvider store={store}>
-      <DummyComponent environmentModifier={environmentModifier} />
-    </StoreProvider>
+    <ErrorBoundary>
```

Comment:
You added the errorBoundary element, but you're not making any assertion against it